### PR TITLE
chore(plugins) Add missing maintainer to css-minimizer-webpack-plugin

### PR DESCRIPTION
**Pull request by montogeek**
_describe your changes..._

- [ ] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [ ] Make sure your PR complies with the [writer's guide][2].
- [ ] Review the diff carefully as sometimes this can reveal issues.
- [ ] Do not abandon your Pull Request: [Stale Pull Requests][3].
- __Remove these instructions from your PR as they are for your eyes only.__

[1]: https://cla.js.foundation/webpack/webpack.js.org
[2]: https://webpack.js.org/contribute/writers-guide/
[3]: https://webpack.js.org/contribute/#pull-requests

### CODE REVIEW

File: src/components/Organization/projects.json [hunk 1]
```diff
@@ -255,6 +255,6 @@
 		"repo": "webpack-contrib/css-minimizer-webpack-plugin",
 		"npm": "css-minimizer-webpack-plugin",
 		"description": "Optimize and minify your CSS using cssnano.",
-		"maintainer": ""
+		"maintainer": "cap-Bernardito"
```
Comment (evilebottnawi) [inline src/components/Organization/projects.json:258]: I think better to use `webpack-contrib` here, because it supported by multiple developers

Comment (montogeek) [reply src/components/Organization/projects.json:258]: Yeah, also this should support an array of contributors

Comment (EugeneHlushko) [reply src/components/Organization/projects.json:258]: ok given its not supported lets merge, it was sufficient previously

**Review by EugeneHlushko — APPROVED** [review]

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**